<ui version="4.0" >
 <class>CalEvents</class>
 <widget class="QWidget" name="CalEvents" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Calendar events</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="ohHeaderLabel" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Select an optional calendar file with the official holidays</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KUrlRequester" name="ohUrlRequester" >
     <property name="filter" >
      <string>*.ics</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ohExplainLabel" >
     <property name="font" >
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:italic;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">You can download a calendar for your country from &lt;a href="http://www.icalshare.com">&lt;span style=" text-decoration: underline; color:#0057ae;">http://www.icalshare.com&lt;/span>&lt;/a> or other sites.&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This is fully optional. All the events from this calendar will be printed in red.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line1" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="fhHeaderLabel" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Select an optional calendar file with your "family holidays"</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KUrlRequester" name="fhUrlRequester" >
     <property name="filter" >
      <string>*.ics</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="fhExplainLabel" >
     <property name="font" >
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text" >
      <string>You can create such a calendar using KOrganizer or any other calendar program.
This is fully optional. All the events from this calendar will be printed in green.</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer1" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>41</width>
       <height>140</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >klineedit.h</include>
  <include location="local" >klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
 <slots>
  <signal>fhChanged(QString)</signal>
  <signal>ohChanged(QString)</signal>
 </slots>
</ui>
